### PR TITLE
Fixed #49: duplicate polls

### LINKED ISSUE #49: Copy/Duplicate polls
Either offer a way to export and import polls or offer a way to duplicate polls so users don't have to replicate the same polls over and over. Copy subscribers also.

### PR DESCRIPTION
**Pull request by asterite**
This adds a button to the UI to duplicate a poll.

screenshot 2016-02-15 11 29 30

When you click it a confirmation appears:

screenshot 2016-02-15 11 30 17

If you click "OK" the poll will be duplicated with a name "{old name} (Copy)", and the browser is redirected to that poll.

screenshot 2016-02-15 11 30 57

From there you can edit the poll.

If you duplicate this new poll, it will have the name "Test Poll (Copy 2)", etc.

### CODE REVIEW

File: app/models/poll.rb [hunk 1]
```diff
@@ -168,6 +168,54 @@ def respondent_question
     self.questions.where(collects_respondent: true).first
   end
 
+  def duplicate
+    duplicate = self.dup
```
Comment (spalladino) [inline app/models/poll.rb]: Should `current_occurrence` be cleared out when duplicating?

Comment (asterite) [inline app/models/poll.rb]: I guess so, yes. I have no idea what that field is :-)

Comment (spalladino) [inline app/models/poll.rb]: That's related to the poll's `recurrence`, which should fire the poll's questions regularly every X days (depending on the recurrence configured). As far as I understand, if the poll hasn't started yet, the `current_occurrence` should be blank. But be sure to double-check.

Comment (asterite) [inline app/models/poll.rb]: Yes, it's nil, just pushed the change.

**Pull request merged**